Сборка начата 4/16/2016 1:41:03 PM.
     1>Проект "C:\Users\Евгений\Desktop\os\lab3\client\lab3-client.vcxproj" в узле 2 (целевые объекты Build).
     1>ClCompile:
         C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\CL.exe /c /Zi /nologo /W3 /WX- /Od /Oy- /D WIN32 /D _DEBUG /D _WINDOWS /Gm /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Fo"Debug\\" /Fd"Debug\vc120.pdb" /Gd /TP /analyze- /errorReport:prompt main.cpp
         main.cpp
     1>main.cpp(18): warning C4297: WinMain: ожидается, что функция не будет выдавать исключения, но она это делает
                 для функции указан атрибут __declspec(nothrow) или throw()
       Link:
         C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\link.exe /ERRORREPORT:PROMPT /OUT:"C:\Users\Евгений\Desktop\os\lab3\client\Debug\lab3-client.exe" /INCREMENTAL /NOLOGO kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib uuid.lib odbc32.lib odbccp32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /DEBUG /PDB:"C:\Users\Евгений\Desktop\os\lab3\client\Debug\lab3-client.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:\Users\Евгений\Desktop\os\lab3\client\Debug\lab3-client.lib" /MACHINE:X86 Debug\main.obj
         lab3-client.vcxproj -> C:\Users\Евгений\Desktop\os\lab3\client\Debug\lab3-client.exe
     1>Сборка проекта "C:\Users\Евгений\Desktop\os\lab3\client\lab3-client.vcxproj" завершена (целевые объекты Build).

Сборка успешно завершена.

Затраченное время: 00:00:00.69
